docs/assets/images/manual/vz330_mellow/electronics/Printer_config/printer.cfg

```diff
@@ -212,10 +212,10 @@ heater_pin: PB0
 sensor_type: PT1000
 sensor_pin: PF4
 
-#control: pid
-#pid_Kp=28.737 
-#pid_Ki=1.935 
-#pid_Kd=106.684
+control: pid
```

Comment:
These values use `pid_xx: value` while the ones below use `pid_xx=value`. I assume this is not correct?